lib/run_argument_parser.py

```diff
@@ -229,3 +229,8 @@ def __init__(self):
             '-n', "--input-n-mer", default='25',
             help="Length of the peptide sequence to use when creating the FASTA from the pVACseq TSV.",
         )
+        self.parser.add_argument(
+            '--spacers', type=lambda s:[spacer for spacer in s.split(',')],
+            help="Comma-separated list of spacers to use for testing junction epitopes. Include an empty string (\"\") to test junctions without spacers.",
```

Comment:
This is really `include two double-quotes in a row`, isn't it?  An empty string would be nothing, like `BEFORE,,AFTER`.  It takes special care to send the quotes along in a shell (as longtime users of UR boolean expression are no doubt all-too-aware :smile:).

Reply:
Might it be better to just use a special string ("None"?) in place of an empty string, since there are so many ways to screw up the quoting? (quotes get eaten by job submissions, scripts, etc)

Reply:
I just pushed a commit to replace the `""` option with `None`. Unfortunately we can't use an empty string because then we wouldn't be able to submit just the no-spacer option.